example/run_hypershard.py

```diff
@@ -23,8 +23,9 @@
 
     if not os.path.isfile(file_name):    
         url_opener = urllib.URLopener()
-        url_opener.retrieve("https://search.maven.org/remotecontent?filepath=com/dropbox/mobile/hypershard/hypershard/1.0.0/hypershard-1.0.0.jar", file_name)
-    hypershard_command = "java -jar hypershard.jar UiTest src/test/resources"
+        url_opener.retrieve("https://oss.sonatype.org/service/local/repositories/snapshots/content/com/dropbox/mobile/hypershard/hypershard/1.0.1-SNAPSHOT/hypershard-1.0.1-20191015.055746-1.jar", file_name)
```

Comment:
the version should come from a variable imo or else you'll forget to update this

Reply:
alternatively add it to the REALEASING.md so that someone can do it after you get hit by a bus ;-)